work around msvcompiler being "removed" in setuptools 74

this is not the right patch in long term but it's the
easiest way to make g-i work again (this module has been
marked deprecated for some years now and was "dropped"
in setuptools 74).

-- q66

--- a/giscanner/ccompiler.py
+++ b/giscanner/ccompiler.py
@@ -26,7 +26,7 @@ import tempfile
 import sys
 import distutils
 
-from distutils.msvccompiler import MSVCCompiler
+from distutils._msvccompiler import MSVCCompiler
 from distutils.unixccompiler import UnixCCompiler
 from distutils.cygwinccompiler import Mingw32CCompiler
 from distutils.sysconfig import get_config_vars
